src/keyszer/config_api.py

```diff
@@ -41,6 +41,29 @@
 # multipurpose timeout
 _TIMEOUTS = TIMEOUT_DEFAULTS
 
+# global dict of delay values used to mitigate Unicode entry sequence and macro or combo failures
+THROTTLE_DELAY_DEFAULTS = {
+    'key_pre_delay_ms': 0,
+    'key_post_delay_ms': 0,
+}
+_THROTTLES = THROTTLE_DELAY_DEFAULTS
```

Comment:
Why two variable names?

Reply:
Just modeling it off of what was existing. It seems nice that the first one makes it clear that it is just establishing the default form of the dict, while the other is a shorter name for actual usage in the rest of the code. 

Did you not create these? 

```py
TIMEOUT_DEFAULTS = {
    "multipurpose": 1,
    "suspend": 1,
    # TODO: not implemented yet
    "post_combo": 0.5,
}

# multipurpose timeout
_TIMEOUTS = TIMEOUT_DEFAULTS
```

Reply:
Ha.  Maybe.  I'm trying to decide now if I actually like that pattern... I think I was trying to separate the defaults from the "live"... that would be important if you ever planned to "reset" the system, but that's not a thing we actually do... anyways since it's a pattern already in the code, we'll run with it as-is. :)

Reply:
I noticed it was more of self-documenting the original value than actually being a useful reference on its own. We can remove that inconsistency with something like `deepcopy()` when assigning to the new variable. That way the default values remain if you use the original reference. Want me to also add it to the timeouts?

Reply:
Apparently this needs "import copy", another built-in Python module. 

```py
import copy
_THROTTLES = copy.deepcopy(THROTTLE_DELAY_DEFAULTS)
```

Reply:
Nah, we don't actually need them... but that's a good point - it's assign by ref so they're destroyed in any case. :)